--- a/configure.ac
+++ b/configure.ac
@@ -15,6 +15,7 @@
 fi
 
 AM_PATH_PYTHON([2.5])
+pythondir="$(cd $PREFIX/Lib/site-packages && pwd -W)"
 
 xcbincludedir='${datadir}/xcb'
 AC_SUBST(xcbincludedir)
diff --git a/xcb-proto.pc.in b/xcb-proto.pc.in
index c7c8b47..690e3e4 100644
--- a/xcb-proto.pc.in
+++ b/xcb-proto.pc.in
@@ -3,9 +3,9 @@ exec_prefix=@exec_prefix@
 datarootdir=@datarootdir@
 datadir=@datadir@
 libdir=@libdir@
-xcbincludedir=${pc_sysrootdir}@xcbincludedir@
+xcbincludedir=@xcbincludedir@
 PYTHON_PREFIX=@PYTHON_PREFIX@
-pythondir=${pc_sysrootdir}@pythondir@
+pythondir=@pythondir@
 
 Name: XCB Proto
 Description: X protocol descriptions for XCB
