diff --git a/configure.ac b/configure.ac
index 9350e71..a01f986 100644
--- a/configure.ac
+++ b/configure.ac
@@ -15,6 +15,7 @@ if test "$XMLLINT" = "no"; then
 fi
 
 AM_PATH_PYTHON([2.5])
+pythondir="$(cd $PREFIX/Lib/site-packages && pwd -W)"
 
 xcbincludedir='${datadir}/xcb'
 AC_SUBST(xcbincludedir)
diff --git a/xcb-proto.pc.in b/xcb-proto.pc.in
index 0822b18..8de18af 100644
--- a/xcb-proto.pc.in
+++ b/xcb-proto.pc.in
@@ -2,9 +2,9 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 datarootdir=@datarootdir@
 datadir=@datadir@
-xcbincludedir=${pc_sysrootdir}@xcbincludedir@
+xcbincludedir=@xcbincludedir@
 PYTHON_PREFIX=@PYTHON_PREFIX@
-pythondir=${pc_sysrootdir}@pythondir@
+pythondir=@pythondir@
 
 Name: XCB Proto
 Description: X protocol descriptions for XCB
